package/yast2-bootloader.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Wed Feb 25 21:48:10 UTC 2015 - jreidinger@suse.com
+
+- Fix creation of multipath device map
```

Comment:
Is there a bug number for that? I'm missing the info in PR description: Which products are affected? None? This is important from maintenance point of view.

So, PLS either add bug number or summarize in PR description. Anyway, good PR description does never hurt anyway.